docs/sdk_python/delete_table.mdx

```diff
@@ -3,6 +3,18 @@ title: Remove a Table
 sidebarTitle: Remove a Table
 ---
 
-<Info>
-This feature is in progress.
-</Info>
+## Description
+
+The `drop_table()` method enables you to delete a table from a connected data source.
+
+## Syntax
+
+Here is the syntax:
+
+```sql
+# option 1
+data_source.drop_table('table_name')
+
+# option 2
+my_table.delete()
```

Comment:
delete and drop are not the same:
delete - is deleting rows in table. 
`db.tables.drop('table2')`
drop - is dropping the whole table
`db.tables.table2.delete(a=1, b=2)`